src/binding/petsc4py/src/petsc4py/PETSc/DMComposite.pyx

```diff
@@ -1,31 +1,80 @@
 # --------------------------------------------------------------------
 
 cdef class DMComposite(DM):
+    """A DM object that is used to manage data for a collection of DMs."""
 
-    def create(self, comm=None):
+    def create(self, comm: Comm | None = None) -> None:
+        """Create a composite object.
+
+        It is used to generate composite vectors made up of several subvectors.
```

Comment:
this is wrong. Just copy paste any other create routine. This creates a DM, not vectors